src/webhook/github.js

```diff
@@ -73,19 +73,80 @@ module.exports = async (req, res) => {
         let author = await getOrCreateUser(prData.user.login);
         if (!author) return res.status(200).send('Failed to get or create user. Skipping process.');
 
-        if (action === 'opened') {
+        const labels = prData.labels;
+        const highestPriorityLabel = labels.reduce((prev, current) => {
```

Comment:
what if no label in pr?

Reply:
@soni-shashan we'll get error 
can not read property 'name' of undefined wherever we've used `highestPriorityLabel`
also instead of default value `""` in label, keep it optional